build/dom_utils.js

```diff
@@ -206,16 +209,58 @@ function fireEvent(eventName, target) {
   target.dispatchEvent(event);
 };
 
+var patchRefs = function patchRefs(refs) {
+  var promises = Object.keys(refs).reduce(function (acc, key) {
+    var $el = refs[key];
+    if ($el instanceof Array) {
+      return acc.concat($el);
```

Comment:
In case of `Array` is returned `acc.concat($el)`, but actually the same concatenation is returned on line 218.
Is this return statement returning the same value despite the `if` at line 215 ?

Reply:
right, this `if` makes no sense